ui/imports/shared/views/HistoryView.qml

```diff
@@ -112,14 +104,15 @@ ColumnLayout {
 
             // LocaleUtils is not accessable from inside expression, but local function works
             property var formatDate: (ms) => LocaleUtils.formatDate(ms, Locale.ShortFormat)
-            sorters: RoleSorter {
-                roleName: "timestamp"
-                sortOrder: Qt.DescendingOrder
+            sorters: ExpressionSorter {
```

Comment:
The sorting is done in SQL query, not at runtime. Now descending by timestamp is activated by default. We have to remove runtime sorting, which is redundant.